graphium/data/dataset.py

```diff
@@ -368,6 +368,8 @@ def __getitem__(self, idx):
             data_dict = self.load_graph_from_index(idx)
             datum["features"] = data_dict["graph_with_features"]
             # these type changes are temporary for ipu dtype match
+            if hasattr(data_dict["labels"], "graph_l1000_mcf7"):
```

Comment:
okay for the experiments now. But please don't introduce this kind of hard-coding into the final code.